f5router/f5router.go

```diff
@@ -819,6 +822,7 @@ func (r *F5Router) fetchExistingDataGroup(c *config.Config, name string) (*bigip
 						break
 					}
 					interval *= 2
+					r.logger.Warn("retry-backoff-encountered-error-retrying", zap.Error(rvs.err))
```

Comment:
keep the interval in here, i think it's good to tell them when the retry will be

Reply:
They'll get that on the next loop iteration see line 807

Reply:
gotcha, that part was collapsed - great